docs/06-manual-builds.md

```diff
@@ -0,0 +1,136 @@
+# Manual builds
+
+If you want to create `.ipa` or `.apk` builds for testing purposes you can follow this guide. If you only want to quickly push an update to the stores then take a look [at the CI/CD documentation](08-cicd.md#triggering-a-build-in-ci).
+
+## Manually building for iOS
+
+First setup XCode and [CocoaPods](#setup-of-cocoapods).
+Now you have to prepare a [few environment variables](08-cicd.md#environment-variables-and-dependencies):
+
+```bash
+export FASTLANE_USER=<secret>
+export FASTLANE_PASSWORD=<secret>
+export MATCH_PASSWORD=<secret>
+```
+
+Setup the certificates locally:
+
+```bash
+cd ios/
+bundle exec fastlane certificates
+```
+
+*Hint: After setting up the certificates you can also star to use XCode to build the app.*
+
+Build the app:
+
+```bash
+yarn
+cd ios/
+bundle exec pod install
+bundle exec fastlane build
+```
+
+Fastlane should report where the build artifacts are. These can be uploaded to App Store Connect.
+
+### Run the app in a simulator
+
+First setup XCode and [CocoaPods](#setup-of-cocoapods).
+If you want to quickly run the app in an emulator do the following:
+
+```bash
+yarn
+cd ios/
+bundle exec pod install
+cd ..
+yarn start
+yarn ios
+```
+
+### Setup of CocoaPods
+
+CocoaPods is used for dependency management of the native libraries.
+First you have to make sure that Ruby is working correctly. We advise to use RVM as mentioned in the [setup of Fastlane](08-cicd.md#setup-of-fastlane). After running this setup you should also be able to do `bundle exec pod --version`.
+
+*Note: `bundle exec pod install` uses the versions from the `Podfile.lock`. `bundle exec pod update` updates the `Podfile.lock`.*
+
+## Manually building for Android
+
+If you do not have Fastlane installed [skip to the next section](#quick-builds-using-a-test-signing-key).
+Firstly you have to prepare a [few environment variables](08-cicd.md#environment-variables-and-dependencies):
+
+```bash
+export CREDENTIALS_GIT_REPOSITORY_URL=<secret>
+export CREDENTIALS_DIRECTORY_PATH=/tmp/credentials
+export CREDENTIALS_KEYSTORE_PASSWORD=<secret>
+export CREDENTIALS_KEYSTORE_PATH=/tmp/credentials/<secret>.enc
+export KEYSTORE_PATH=/tmp/keystore.jks
```

Comment:
I think it would be better not having to define environment variables but rather passing them as arguments to fastlane commands.

Reply:
I created https://issues.integreat-app.de/browse/NATIVE-548 instead